code/datums/traits/good.dm

```diff
@@ -108,3 +108,19 @@
 	mob_trait = TRAIT_VORACIOUS
 	gain_text = "<span class='notice'>You feel HONGRY.</span>"
 	lose_text = "<span class='danger'>You no longer feel HONGRY.</span>"
+
+
+
+/datum/quirk/hlumi_eyes
+	name = "High-Luminosity Eyes"
+	desc = "Your eyes have been cybernetically enhanced, high-luminosity allow you to change your eye color at a moment's notice and shine light away from yourself"
+	value = 1
+
+/datum/quirk/hlumi_eyes/on_spawn()
+	var/mob/living/carbon/human/H = quirk_holder
+	var/obj/item/organ/eyes/robotic/glow/hlumi = new
+	hlumi.Insert(H, drop_if_replaced = FALSE)
+	H.regenerate_icons()
+
+/datum/quirk/hlumi_eyes/post_add()
+	to_chat(quirk_holder, "<span class='boldnotice'>Your eyes have been enhanced with cybernetic high-luminosity eyes.</span>")
```

Comment:
use gain_text, no need to overwrite this proc.

Reply:
![image](https://user-images.githubusercontent.com/25509881/39967472-eddefc44-56bb-11e8-8578-3beabe28de42.png)
I based that on prosthetic arm trait

Reply:
it uses it in `post_add` because of `[slot_string]`